RecoMuon/Configuration/python/RecoMuonPPonly_cff.py

```diff
@@ -11,20 +11,23 @@
 
 # Displaced SA muons
 from RecoMuon.MuonSeedGenerator.CosmicMuonSeedProducer_cfi import *
-displacedMuonSeeds = CosmicMuonSeed.clone()
-displacedMuonSeeds.ForcePointDown = False
+displacedMuonSeeds = CosmicMuonSeed.clone(
+    ForcePointDown = False
+)
 
-displacedStandAloneMuons = standAloneMuons.clone()
-displacedStandAloneMuons.InputObjects = cms.InputTag("displacedMuonSeeds")
-displacedStandAloneMuons.MuonTrajectoryBuilder = cms.string("StandAloneMuonTrajectoryBuilder")
-displacedStandAloneMuons.TrackLoaderParameters.VertexConstraint = cms.bool(False) 
+displacedStandAloneMuons = standAloneMuons.clone(
+    InputObjects = 'displacedMuonSeeds',
+    MuonTrajectoryBuilder = 'StandAloneMuonTrajectoryBuilder'
+)
+displacedStandAloneMuons.TrackLoaderParameters.VertexConstraint = False 
```

Comment:
The following should work:
```suggestion
displacedStandAloneMuons = standAloneMuons.clone(
    InputObjects = 'displacedMuonSeeds',
    MuonTrajectoryBuilder = 'StandAloneMuonTrajectoryBuilder',
    TrackLoaderParameters = dict(VertexConstraint = False)
)
```